### DISCUSSION TITLE
Groups association: motion sensor that triggers a switch

### DISCUSSION THREAD

**Discussion by roblomq** [zwave-js/zwave-js-ui#2715] [General]
I'm trying to let a Fibaro FGMS001 motion sensor switch on and off a Fibaro FGS222 Double switch. But as i'm not familiar with the groups function hopefully someone is willing to help me in how to achieve this. Or maybe there is a good manual, I've searched it but cannot find anything.

What I have is the situation below, the groups are automatically created. I can add something to the switch but the motion sensor doesn't seem to change anything also not when I try to wake up the device.

![motion to switch](https://user-images.githubusercontent.com/59142034/194773006-707aac0d-9051-452d-b82b-2251faf53a6d.jpg)

**Comment by robertsLando** [comment]
@roblomq  could be useful to know the actual groups available on your motion sensor. Go to debug tab of your node and attach content here

cc @blhoward2 @kpine

**Reply by kpine** [reply]
Last I recall the debug output doesn't include the supported association groups, only the assigned ones. Still true?

**Reply by roblomq** [reply]
I have no experience with this and I did some trial and error which led probably to this current situation. 
I'm missing some knowledge and was not able to find this documented somewhere so hopefully someone can give done explanation on how to create an association.

**Reply by roblomq** [reply]
I think it's about the code below. This was something that was already in before I started to experiment with it.

> "groups": [
    {
      "text": "Motion Sensor Status",
      "endpoint": 0,
      "value": 1,
      "maxNodes": 5,
      "isLifeline": false,
      "multiChannel": true
    },
    {
      "text": "Tamper Alarm",
      "endpoint": 0,
      "value": 2,
      "maxNodes": 5,
      "isLifeline": true,
      "multiChannel": true
    },
    {
      "text": "Controller Updates",
      "endpoint": 0,
      "value": 3,
      "maxNodes": 1,
      "isLifeline": true,
      "multiChannel": false
    }
  ],

How can I revert this back to default? Or is this default?

**Reply by robertsLando** [reply]
> Last I recall the debug output doesn't include the supported association groups, only the assigned ones. Still true?

Nope, `groups` contains all available groups

**Comment by roblomq** [comment]
I thought I went crazy, but on a mobile device the debug is not available...
Anyway, in the attachment the output of the debug of the motion sensor.

[Motion debug.txt](https://github.com/zwave-js/zwave-js-ui/files/9747383/Motion.debug.txt)

**Comment by bvdf0251** [comment]
Adding new life to this as I'm looking for the exact same thing.

There's a couple more groups available now with new firmware. There's a group 4 for motion sensor that can be set to a target node. But when I try to add the group it immediately fails.

Setting the same association on an Aeotec wallmote does work, so the association can work.

But it just doesn't want to get set on the Fibaro motion sensor.

![Screenshot_20240403_132051_Chrome](https://github.com/zwave-js/zwave-js-ui/assets/108339342/9606e1b6-c08e-4868-b093-dbbfd68cbd2a)
![Screenshot_20240403_132028_Chrome](https://github.com/zwave-js/zwave-js-ui/assets/108339342/93e9e496-6847-40b6-9b5e-ef13fbbe4526)

**Reply by kpine** [reply]
Make sure source and target nodes are using the same Security (S0 vs S2) and same security classes (e.g. S2 Authenticated + S2 Authenticated). The ZUI logs probably have more details.

**Reply by bvdf0251** [reply]
They are all at S0. Creating the same association on aeotec wallmote to Fibaro switch is working as expected.

Suspect the motion sensor is not awake for the correct amount of time. Or is just can't manually create association.

Tried deleting group 1 to controller and assign it to the switch also no go. 

Reintevriew has set it back to controller.

I have zwave js ui setup with mqqtt and domoticz so turning the light on via automation/on motion is working but it has about 3 second delay between motion sensor first blink and ligt actually turning on.

**Comment by bvdf0251** [comment]
Removed S0 from both devices. Put them in without security and its way better now. Almost instantly. Good enough for the WAF.

**Comment by sergiobaiao** [comment]
hi there, i'm on a very similar situation here. Just added my FGMS001 to ZWaveJS/HA and it behaving weirdly. First, the FGMS001 Motion appears as a "Basic" binary switch on HA instead of a binary sensor. Second, when i associate the FGMS001 to the corresponding light switch, everytime it's activated the light starts blinking. Did you find anything to solve this?
